payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SectionUI.kt

```diff
@@ -20,35 +20,44 @@ import androidx.compose.ui.unit.Dp
 import androidx.compose.ui.unit.TextUnit
 import androidx.compose.ui.unit.dp
 import androidx.compose.ui.unit.sp
+import com.stripe.android.ui.core.StripeTheme
 
 /**
  * This is the style for the section card
  */
-internal data class CardStyle(
-    private val isDarkTheme: Boolean,
-    val cardBorderColor: Color = if (isDarkTheme) {
-        Color(0xFF787880)
-    } else {
-        Color(0x14000000)
-    },
-    val cardBorderWidth: Dp = 1.dp,
-    val cardElevation: Dp = 0.dp,
+
+internal object CardStyle {
+    val cardBorderColor: Color
+        @Composable
+        @ReadOnlyComposable
+        get() = StripeTheme.colors.colorComponentBorder
+
+    val cardDividerColor: Color
+        @Composable
+        @ReadOnlyComposable
+        get() = StripeTheme.colors.colorComponentDivider
+
+    val cardBorderWidth: Dp = 1.dp
+    val cardElevation: Dp = 0.dp
     val cardStyleBackground: Color = Color(0x20FFFFFF)
```

Comment:
This background color is only used in Link components.

Reply:
Can you explain where this is used? And what's used in Payment Sheet instead? Can Link just set the value it wants like Payment Sheet will do?

Reply:
It turns out this is used in `SectionCard` to declare a background that is overridden by child views in both link use cases and the one for case for payment sheet. I changed it to a few different colors and didn't observe any changes. I removed it.